client.go

```diff
@@ -64,8 +64,9 @@ type Client struct {
 
 	Log zerolog.Logger
 
-	RequestHook  func(req *http.Request)
-	ResponseHook func(req *http.Request, resp *http.Response, duration time.Duration)
+	RequestHook       func(req *http.Request)
+	ResponseHook      func(req *http.Request, resp *http.Response, duration time.Duration)
+	ResponseErrorHook func(req *http.Request, resp *http.Response, err error, duration time.Duration)
```

Comment:
The error parameter should probably just be added to ResponseHook

Reply:
Do we care about breaking the API?

Reply:
Not really, I don't even remember what the hooks were added for